infra/base-images/base-builder/Dockerfile

```diff
@@ -133,14 +133,29 @@ RUN cd $SRC && \
     tar -xzv --strip-components=1 -f $SRC/oss-fuzz.tar.gz && \
     rm -rf examples $SRC/oss-fuzz.tar.gz
 
+RUN cd $SRC && \
+    git clone \
+      --depth 1 \
+      --branch main \
```

Comment:
are `--branch main --single-branch` necessary? Are the quotes necessary? Is `centipede` at the end necessary (for sure not).
Remove extra unneeded stuff please.